crates/nu-cli/src/shell/help_shell.rs

```diff
@@ -230,56 +229,3 @@ impl Shell for HelpShell {
         ))
     }
 }
-
-impl completion::Completer for HelpShell {
```

Comment:
I decided to drop this from `HelpShell` and `ValueShell`, but these will eventually come back. I think a better option is to build a generic "path completer" that makes use of a `Shell`'s `ls` method.